### DISCUSSION TITLE
Sawirada

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#81] [General]
Dhalinyaro sawirada nala soo wadaaga waliba **sawirka** oo **high quality** oo `zip` ah. `zip` intaad kaso dhigtaan isodira si aanu u lumin tayada sawirka Insha'Allah!

@dugsiiyeinc/dhaqsopay-extension-developers 
@dugsiiyeinc/dhaqsopay-server-developers 
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by Nasrah-muse** [comment]
Halkaan miyaa lasoo dhigayaa eng

**Reply by sharafdin** [reply]
maya. zip ahaan igu soodira aniga Insha'Allah!
